### PR TITLE
Multiplatform-demo example

### PR DESCRIPTION
**Pull request by igordmn**
1. Make MPP example as a main example for Compose Multiplatform: 
image

2. For iOS/Android/Web it will run as a list of examples, for Desktop we create separate configurations to speedup opening (for other targets we will figure something in the future probably):

image

### CODE REVIEW

File: compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/Example1.kt [hunk 1]
```diff
@@ -0,0 +1,138 @@
+package androidx.compose.mpp.demo
```
Comment (igordmn) [inline compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/Example1.kt:1]: This file is a copy of the previous App.kt, just with renamed main function

**Review by MatkovIvan — CHANGES_REQUESTED** [review]
A new `demo.ImageViewer` target fails on start with
```
Exception in thread "main" java.lang.NullPointerException
	at androidx.compose.mpp.demo.App.<init>(App.kt:39)
```

**Comment by igordmn** [comment]
> A new demo.ImageViewer target fails on start with

Removed it in this PR, will add in another PR

**Comment by igordmn** [comment]
cc @dima-avdeev-jb. This PR changes the main mpp example for iOS. now we have a list of examples

**Review by MatkovIvan — APPROVED** [review]

**Comment by igordmn** [comment]
> cc @dima-avdeev-jb. This PR changes the main mpp example for iOS. now we have a list of examples

Actually, I forgot to change the other targets. Now it is the main example.

Wasn't able to remove demo-uikit module, which we don't need right now.

Added Dima as a reviewer too

File: compose/mpp/demo-uikit/README.md [hunk 2]
```diff
@@ -1,9 +1,4 @@
-### Setup
- - in ~/.gradle/gradle.properties set: 
-```
-androidx.compose.multiplatformEnabled=true
-```
- - Open dir .../frameworks/support with AppCode, install plugin KMM for AppCode.
+ - Open dir ../frameworks/support with Android Studio, install KMM plugin.
```
Comment (dima-avdeev-jb) [inline compose/mpp/demo-uikit/README.md]: We still need AppCode for now to launch this demo.
Or use `xcodegen` tool in mpp/demo and launch in Xcode.

**Comment by dima-avdeev-jb** [comment]
Thanks, I checked:
 - in AppCode
 - in Xcode with `xcodegen` tool

It works!

File: compose/mpp/demo-uikit/README.md (hunk 2 above)
Comment (dima-avdeev-jb) [reply compose/mpp/demo-uikit/README.md]: I suggest to revert this line

Comment (igordmn) [reply compose/mpp/demo-uikit/README.md]: Done

**Review by dima-avdeev-jb — APPROVED** [review]

**Pull request merged**